### DISCUSSION TITLE
第4节课(20班)

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#390] [作业]
宝子们，20班哦

**Comment by deyouzuishuai** [comment]
https://zhuanlan.zhihu.com/p/679190962

**Comment by liu673** [comment]
https://github.com/liu673/InternLM_OpenNotebook/blob/master/lesson_4/lesson_4_homework.md

**Comment by 20300240011** [comment]
https://zhuanlan.zhihu.com/p/679645861

**Comment by LiyanJin** [comment]
https://blog.csdn.net/weixin_53443275/article/details/135857581
